tests/pipeline/test_engine.py

```diff
@@ -112,6 +111,16 @@ def compute(self, today, assets, out, open):
         out[:] = open
 
 
+class MultipleOutputs(CustomFactor):
+    window_length = 1
```

Comment:
Do we have a test that runs a multi-output factor with a nontrivial window_length?